pyUltroid/fns/tools.py

```diff
@@ -432,19 +438,142 @@ async def get_google_images(query):
     random.shuffle(google_images)
     return google_images
 
+###############################################################################
```

Comment:
again, a whole load of credits to myself. why?

Reply:
In the event I made my repo public, it's basically your bot.